modules/react/src/components/export-video/export-video-panel-container.js

```diff
@@ -166,6 +174,7 @@ export class ExportVideoPanelContainer extends Component {
         }
         // TODO Add other encoders as needed. Not yet implemented
       }
+      // adapter: new DeckAdapter(this.getDeckScene)
```

Comment:
Is this meant to be commented out now?

Reply:
Yes, it's supposed to be commented out. This line isn't part of a working commit. It was something I tried to get the modal preview to update. However, now I know that it's related to requestAnimationFrame

This is the error you'll get when it's enabled. Loses track of the canvas that was set in this.state
![image](https://user-images.githubusercontent.com/33266041/99205694-1f4da480-276e-11eb-8db6-f2b5aa9b7c75.png)